tests/phpunit/includes/testcase-trait.php

```diff
@@ -4,17 +4,48 @@
  * A trait to share code between several test case classes.
  */
 trait PLL_UnitTestCase_Trait {
+
+	/**
+	 * @var array|null
+	 */
+	protected $options;
+
+	/**
+	 * @var PLL_Links_Model|null
+	 */
+	protected $links_model;
+
+	/**
+	 * @var PLL_Model|null
+	 */
+	protected $pll_model;
+
+	/**
+	 * @var PLL_Frontend|null
+	 */
+	protected $frontend;
+
+	/**
```

Comment:
`$pll_env` and `$pll_admin` are often the same given that `PLL_Admin` is a child class of `PLL_Base`. Should we merge these two into one ? And add another `$pll_something` for special environment (like `PLL_Frontend`) ?

Reply:
I'm not sure to understand what you mean.
From what I see, `$pll_env` can be:
- In PLL: `PLL_Frontend`, `PLL_Settings`.
- In PLL Pro: `PLL_Frontend`, `PLL_Settings`, `PLL_Admin`, `PLL_REST_Request`.